build-system/pr-check/bundle-size-module-build.js

```diff
@@ -18,7 +18,9 @@ const jobName = 'bundle-size-module-build.js';
  * Steps to run during push builds.
  */
 function pushBuildWorkflow() {
-  timedExecOrDie('amp dist --noconfig --esm --version_override 0000000000000');
+  timedExecOrDie(
+    'amp dist --noconfig --esm --version_override 0000000000000 --nomanglecache'
```

Comment:
nit: it could make sense to instead make a new flag `--for_bundle_size` which encapsulates both of these flags

Reply:
Mixed feelings about this. I think being explicit about what each flag does is better than saying "build this for a specific use-case", but I also don't really know what "mangle cache" is